src/options.ts

```diff
@@ -96,44 +99,30 @@ hljs.registerLanguage('plaintext', plaintext);
       'ul',
       'var',
     ],
-    allowedAttributes: {'*': ['rel', 'alt', 'crossorigin', 'preload']},
+    allowedAttributes: {'*': ['rel', 'alt', 'crossorigin', 'preload', 'href']},
```

Comment:
Should we add "src" too, because the img and audio tags need them?

Reply:
Ah good call